stubs/app/Models/ResourceCollection.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Translatable\HasTranslations;
+
+class ResourceCollection extends Model
+{
+    use HasFactory;
+    use HasTranslations;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<string>
+     */
+    protected $fillable = [
+        'title',
+        'user_id',
+        'description',
+    ];
+
+    /**
+     * The attributes that are transterms
+     *
+     * @var array
+     */
+    public $translatable = [
+        'title',
+        'description',
+    ];
+
+    /**
+     * Get user of this resource collection
+     */
+    public function user()
```

Comment:
It would be good to add a return type to the function and it's documentation block — you can see what I mean on lines 111-114 of the User model stub.